src/esy/buildEjectCommand/index.js

```diff
@@ -15,6 +15,8 @@ const outdent = require('outdent');
 const {flattenArray} = require('../Utility');
 
 const RUNTIME = fs.readFileSync(require.resolve('./runtime.sh'), 'utf8');
+const fastReplaceString = fs.readFileSync(require.resolve('../../../node_modules/.bin/fastreplacestring'));
```

Comment:
I like how `FastReplaceString` is in its own package, but could we read that `.c` file, and spit it out to be recompiled just like we do for `realpath.c` so that it can be ejected on host?